test/reciperepo-test.js

```diff
@@ -1,13 +1,222 @@
+        {
+          id: 302,
+          quantity: {
+            amount: 10,
+            unit: 'cup'
+          }
+        },
+        {
+          id: 410,
+          quantity: {
+            amount: 3,
+            unit: 'tbs'
+          }
+        }
+      ],
+      instructions: [
+        { instruction: 'step 1', number: 1 },
+        { instruction: 'step 2', number: 2 },
+        { instruction: 'step 3', number: 3 }
+      ],
+      name: 'Beef Noodle',
+      tags: ['noodles', 'main dish', 'hot dish'],
+    };
+    spaghetti = {
+      id: 1234,
+      image: "https://spoonacular.com/recipeImages/991136-556x370.jpg",
+      ingredients: [
+        {
+          id: 1001,
+          quantity: {
+            amount: 0.25,
+            unit: 'cup'
+          }
+        },
+        {
+          id: 91,
+          quantity: {
+            amount: 12,
+            unit: 'unit'
+          }
+        },
+      ],
+      instructions: [
+        { instruction: 'step 1', number: 1 },
+        { instruction: 'step 2', number: 2 },
+        { instruction: 'step 3', number: 3 }
+      ],
+      name: 'Spaghetti',
+      tags: ['lunch', 'main course', 'main dish', 'dinner']
+    };
+    ingredientData = [
+      {
+        id: 98871,
+        name: 'hawaiian sweet rolls',
+        estimatedCostInCents: 142,
+      },
+      {
+        id: 18372,
+        name: 'bicarbonate of soda',
+        estimatedCostInCents: 582,
+      },
+      {
+        id: 1123,
+        name: 'eggs',
+        estimatedCostInCents: 472,
+      },
+      {
+        id: 1001,
+        name: 'butter',
+        estimatedCostInCents: 1,
+      },
+      {
+        id: 2,
+        name: 'chocolate',
+        estimatedCostInCents: 200,
+      },
+    ];
+  })
+
+  describe('Properties', () => {
+
+    it('should be a function', () => {
+      expect(RecipeRepo).to.be.a('function');
+    });
+
+    it('should be an instance of RecipeRepo', () => {
+      expect(recipe).to.be.an.instanceof(RecipeRepo);
+    });
+
+    it('should have no recipe(s) by default', () => {
+      expect(recipe.recipes).to.deep.equal([]);
+    });
+
+    it('should be able to hold a recipe if passed in', () => {
+      recipe = new RecipeRepo([buffaloChicken]);
+      expect(recipe.recipes).to.deep.equal([buffaloChicken]);
+    });
+
+    it('should be able to hold mutiple recipes', () => {
+      recipe = new RecipeRepo([buffaloChicken, beefNoodle]);
+      expect(recipe.recipes).to.deep.equal([buffaloChicken, beefNoodle]);
+    });
+
+    it('should create a Recipe instance from a recipe passed in', () => {
+      recipe = new RecipeRepo([buffaloChicken]);
+      expect(recipe.recipes[0]).to.be.an.instanceof(Recipe);
+    });
+
+    it('should be able to create multiple Recipe instances', () => {
+      recipe = new RecipeRepo([buffaloChicken, beefNoodle]);
+      expect(recipe.recipes[0]).to.be.an.instanceof(Recipe);
+      expect(recipe.recipes[1]).to.be.an.instanceof(Recipe);
+    });
+  })
```

Comment:
I feel like this part of the test might be redundant. Instead, you could add an extra expect to the test above on line 145 like: 
expect(recipe.recipes[1].to.be.an.instanceof(Recipe);

Reply:
Yeah very true. I'll get rid of the first test and just reword the test that tests 2 instances of the Recipe class